src/index.js

```diff
@@ -134,6 +136,8 @@ class TargetCoreAPI {
   * Creates a new XT activity with the specified contents and returns the created activity.
   *
   * @param body {Object} Activity JSON.
+  * @param options {Object} sdk options
+  * @param options.headers headers to pass to API call
```

Comment:
`options.headers` is missing a type (similarly for the other JSDocs)

Looks like this lib is not using https://github.com/adobe/eslint-config-aio-lib-config which will catch these issues.

Reply:
done